CRM/CiviAwards/Setup/ProcessAwardsCategoryForCustomGroupSupport.php

```diff
@@ -22,7 +22,10 @@ public function apply() {
       self::AWARDS_CATEGORY_CG_LABEL,
       'CRM_CiviAwards_Helper_CaseTypeCategory::getSubTypes;'
     );
-    $caseCategoryCustomData->create(CaseTypeCategoryHelper::AWARDS_CASE_TYPE_CATEGORY_NAME);
+    $caseCategoryCustomData->create([
+      'name' => CaseTypeCategoryHelper::AWARDS_CASE_TYPE_CATEGORY_NAME,
+      'label' => CaseTypeCategoryHelper::AWARDS_CASE_TYPE_CATEGORY_NAME,
+    ]);
   }
```

Comment:
Looks like this method only accepts a single parameter of the case category name and not an array. Can you double check?

Reply:
@tunbola thanks a lot.

I noticed that when creating the menu we use the case category name instead of the label. Probably the reason why we limit the label used. Wanted to have a middle ground in order to fix that issue in the future, but realised all the other extensions depended on this and went back. Should have updated this part.